style.css

```diff
@@ -0,0 +1,1358 @@
+  display: flex;
+  margin-top: 5vh;
+  background-color: #e5e5e5;
+  height: 1px;
+  width: 85vw;
+  margin-left: 4vw;
+}
+
+.main-image {
+  display: flex;
+  margin-top: 10vh;
+  margin-left: 6.5vw;
+  width: 86vw;
+}
+
+.card-1 {
+  display: flex;
+  flex-direction: column;
+  margin-left: 6.5vw;
+  width: 85.5vw;
+}
+
+.multi-story {
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #14284b;
+}
+
+.p21 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+ul.skills {
+  display: flex;
+  flex: 1;
+  list-style: none;
+  margin-top: 2vh;
+}
+
+.css-1 {
+  padding: 2vw;
+  border: 2px solid #8993a4;
+}
+
+.css-1 :hover {
+  border: solid blue;
+}
+
+.skill {
+  margin-left: 6vw;
+  padding: 2vw;
+  border: 2px solid #8993a4;
+}
+
+.skill :hover {
+  border: solid blue;
+}
+
+.see-project-1 {
+  width: 92px;
+  background: #ff6b00;
+  color: #fff;
+  padding: 10px;
+  margin-left: 30px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 24px;
+  text-decoration: none;
+  margin-top: 18px;
+  margin-bottom: 5vh;
+}
+
+.see-project-1 :hover {
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
+/* Profesional Art Printing Data */
+
+.card-2 {
+  display: flex;
+  margin-left: 24px;
+  margin-right: 24px;
+  flex-direction: column;
+  background-image: url(./assets/images/Placeholder1.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 88vw;
+}
+
+.prof-art {
+  margin-top: 17vh;
+  margin-left: 6px;
+  color: #fff;
+  font-family: "Crete Round", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.p22 {
+  padding: 2vw;
+  margin-left: 6px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+ul.skils {
+  display: flex;
+  width: 65vw;
+  margin-bottom: 3vh;
+}
+
+.see-project-2 :hover {
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
+.skil :hover {
+  border: solid blue;
+}
+
+ul.skils :first-child {
+  display: flex;
+  align-self: center;
+  padding: 2vw;
+  border: 2px solid #8993a4;
+  background: rgba(255, 255, 255, 0.24);
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+  margin-left: 15px;
+}
+
+ul.skils :first-child :hover {
+  border: solid blue;
+}
+
+.skil {
+  margin-left: 6vw;
+  padding: 2vw;
+  border: 2px solid #8993a4;
+  background: rgba(255, 255, 255, 0.24);
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.see-project-2 {
+  /* padding: 3vw; */
```

Comment:
- [ ] Kindly remove unnecessary comments from the codebase to make the code more clear and understandable.